README.md

```diff
@@ -25,84 +19,168 @@
+
+During several workshops organized by [NASA Openscapes](https://nasa-openscapes.github.io/events.html) the need to provide easy-to-use tools to our users became evident. We shouldn’t need to be cloud infrastructure engineers to easily work with the data if it’s hosted on a S3 bucket. This was the main reason behind *earthaccess*, a Python library that aims to simplify our workflows to address some of the pain points of programmatic access to NASA datasets within the PyData ecosystem.
+
+With *earthaccess* we can login with NASA, search and download data with a few lines of code and even more relevant, our code will work the same way if we are running it in the cloud or from our laptop. ***earthaccess*** handles authentication with [NASA's Earthdata Login (EDL)](https://urs.earthdata.nasa.gov), search using NASA's [CMR](https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html) and access through [`fsspec`](https://github.com/fsspec/filesystem_spec). If all of this sounds like gibberish... no worries we are here to help! Please ask us anything on [project board!](https://github.com/nsidc/earthdata/discussions).
+
+The only requirement to use this library is to open a free account with NASA [EDL](https://urs.earthdata.nasa.gov).
+
+
+
 
-## Installing earthdata
 
-Install the latest release:
+> **⚠️ Warning ⚠️**: The project has recently been renamed from earthdata to earthaccess. If your environment depends on earthdata, please update your reference to earthaccess v0.5.0
+
+
+
+<a href="https://urs.earthdata.nasa.gov"><img src="https://auth.ops.maap-project.org/cas/images/urs-logo.png" /></a>
+
+<img src="https://user-images.githubusercontent.com/717735/199083271-79ce5113-b680-4fe5-a1bf-6c927bf02c75.jpg" />
+
+## Installing earthaccess
+
+Install the latest release using conda
 
 ```bash
-conda install -c conda-forge earthdata
+conda install -c conda-forge earthaccess
 ```
 
-Or you can clone `earthdata` and get started locally
+Using Pip
 
 ```bash
+pip install earthaccess
+```
 
-# ensure you have Poetry installed
-pip install --user poetry
+Try it in your browser without installing anything! [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/nsidc/earthdata/main)
 
-# install all dependencies (including dev)
-poetry install
+## Do where do we start?
 
-# develop!
-```
+Data Pathfinders guide users through the process of selecting application-specific datasets and learning how to use them through intuitive tools, facilitating equal and open access to the breadth of NASA Earth science data. The pathfinders are intended to familiarize users with and provide direct links to the applicable, commonly-used datasets across NASA’s Earth science data collections. After getting started here, there are numerous NASA resources that can help develop one’s skills further.
+
+[https://www.earthdata.nasa.gov/learn/pathfinders](
+https://www.earthdata.nasa.gov/learn/pathfinders)
```

Comment:
I suggest removing this section, based on the 'Art of the readme' where they recommend making it as short as it needs to be. I think this section makes it a bit too long and might confuse users, I think it is best just to focus on earthaccess instead of linking to other datasets and tools.